.github/ISSUE_TEMPLATE/lwkd-report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Feature an update for LWKD
+about: Submit a news, PR, content or update for the next 'Last Week in Kubernetes Development' Newsletter
+title: 'Add to LWKD - [Title]'
+labels: 'kind/feature'
+---
+
+**Request to add your PR/updates/content/news in the next week's LWKD Newsletter**
+<!-- Please only use this template for submitting content you'd like to feature in the LWKD Newsletter -->
+<!-- See https://kubernetes.io/docs/contribute/start/ for guidance on writing an actionable issue description. -->
+
+**What would you like to be added**
+<!-- Please provide a detailed description of how this news, feature, or enhancement should function from the reader's perspective -->
+
+
+**Why is this needed**
+<!-- Provide more context of how this information can be used in practice?, who will benefit from this?, why are they being added? -->
+
```

Comment:
These three sections should be collapsed down to:

**Your Item for LWKD**

Please write *one sentence* with your news item for LWKD.    This line should very briefly explain what the item is and why it's important.  

**Links for Your Item**

Share one or two links for this item.  Links to merged pull requests and/or Github issues are better than links to mailing lists or web pages.